resfulservice/src/graphql/resolver/material_template/mutation.js

```diff
@@ -30,6 +30,24 @@ const materialMutation = {
     } catch (error) {
       return errorFormater(error.message, 500);
     }
+  },
+
+  deleteXlsxCurationList: async (_, { input }, { user, req, isAuthenticated }) => {
+    req.logger.info('deleteMaterialColumn Function Entry:');
+    if (!isAuthenticated) {
+      req.logger?.error('[deleteMaterialColumn]: User not authenticated to remove listing');
+      return errorFormater('not authenticated', 401);
+    }
+    const { field } = input;
+    try {
+      const columnExists = await MaterialTemplate.findOne({ field });
```

Comment:
I'll suggest removing this. To avoid making it an expensive operation. The findOneAndDelete mongoose function below it should be sufficient.